helios-client/src/main/java/com/spotify/helios/common/descriptors/Job.java

```diff
@@ -633,6 +654,16 @@ public Builder setNetworkMode(final String networkMode) {
       return this;
     }
 
+    public Builder setMetadata(final Map<String, String> metadata) {
+      p.metadata = metadata;
+      return this;
+    }
+
+    public Builder addMetadata(String name, String value) {
```

Comment:
...so that cases where you call `setMetadata` with an immutable map and then call `addMetadata` will still work.